nwb_conversion_tools/gui/nwb_conversion_gui.py

```
@@ -649,15 +657,24 @@ def __init__(self, parent):
 
     def run(self):
         #try:
-        mod_file = self.parent.conversion_module_path
-        spec = importlib.util.spec_from_file_location(os.path.basename(mod_file).strip('.py'), mod_file)
-        conv_module = importlib.util.module_from_spec(spec)
-        spec.loader.exec_module(conv_module)
-        metadata = self.parent.read_metadata_from_form()
-        conv_module.conversion_function(source_paths=self.parent.source_paths,
-                                        f_nwb=self.parent.lin_nwb_file.text(),
-                                        metadata=metadata,
-                                        **self.parent.kwargs_fields)
+        if not self.parent.lin_nwb_file.text():
+            raise ValueError('select a save location for nwbfile')
+        if self.parent.conversion_module_path:# if not an empty string (if value was selected from gui)
+            mod_file = self.parent.conversion_module_path
+            spec = importlib.util.spec_from_file_location(os.path.basename(mod_file).strip('.py'), mod_file)
+            conv_module = importlib.util.module_from_spec(spec)
+            spec.loader.exec_module(conv_module)
+            metadata = self.parent.read_metadata_from_form()
+            conv_module.conversion_function(source_paths=self.parent.source_paths,
+                                            f_nwb=self.parent.lin_nwb_file.text(),
+                                            metadata=metadata,
+                                            **self.parent.kwargs_fields)
+        else:
+            metadata = self.parent.read_metadata_from_form()
```

If the user inputs both a module and a class, that should throw an error.